tasks/Task.cpp

```diff
@@ -93,13 +93,17 @@ bool Task::configureHook()
     cVb.buildFrom(cMb);
     task.set_cVe(cVb);
 
+    //set the time for trigger the timeout
+    timeout = base::Timeout(base::Time::fromSeconds(1));
```

Comment:
Doesn't every single component in the control chain already has this behaviour. Including the velocity controller ?

Does it really make sense to add the same boilerplate complexity to all the control-related components ?